frontend/src/components/Confirm/Confirm.tsx

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/react';
+import { COLOR } from 'src/constants';
+import Card from '../@shared/Card/Card';
+import Heading from '../@shared/Heading/Heading';
+import { Wrapper, Content, ButtonWrapper } from './Confirm.styles';
+
+interface Props {
+  isOpened: boolean;
+  message: string;
+  subMessage?: string;
+  onConfirm: () => void;
+  onCancel: () => void;
+  direction?: 'default' | 'reverse';
+}
+
+const Confirm = ({
+  isOpened,
+  message,
+  subMessage,
+  onConfirm,
+  onCancel,
+  direction = 'default',
+}: Props) => {
+  return (
+    <Wrapper isOpened={isOpened}>
+      <Card width="90%">
+        <Content>
+          <Heading.level3
+            color={COLOR.BLACK}
+            css={css`
+              margin-bottom: 1rem;
+
+              text-align: center;
+
+              line-break: auto;
+              word-break: keep-all;
+            `}
+          >
+            {message}
+          </Heading.level3>
+          {!!subMessage && <p>{subMessage}</p>}
```

Comment:
```suggestion
          {subMessage && <p>{subMessage}</p>}
```

이렇게쓰지 않고 boolean 값으로 쓴 이유가 있을까요?

Reply:
subMessage로 되었을 때 0 과 같은 값이 들어오면 없는 판정이 아니라, 0이 그대로 그려지는 경우가 종종 있어서 이렇게 처리했는데, subMessage 는 string | undefined 타입이라서 크게 필요하지 않을 것 같습니다. 그러나 비교할 때 truthy, falsy 한 값 보다 boolean 값으로 정확하게 사용하고자 다음과 같이 처리했습니다.!